xds/build.gradle

```diff
@@ -66,6 +66,7 @@ sourceSets {
             srcDir 'third_party/envoy/src/main/proto'
             srcDir 'third_party/protoc-gen-validate/src/main/proto'
             srcDir 'third_party/udpa/src/main/proto'
+            srcDir 'third_party/googleapis/src/main/proto'
```

Comment:
@voidzcy Shall we keep the alphabetical order of this?

Reply:
Doesn't matter.

Reply:
OK, SG!